com.unity.render-pipelines.universal/Runtime/RendererFeatures/DecalRendererFeature.cs

```diff
@@ -147,6 +147,7 @@ private void OnDecalMaterialChange(DecalProjector decalProjector)
     }
 
     [DisallowMultipleRendererFeature("Decal")]
+    [Tooltip("Decals.")]
```

Comment:
Decals was added after Theo's tooltip witch hunt so it was missed. I propose this
"The decals renderer feature injects the render passes required for decals to render."

Reply:
@TheoWong-pixel @oleks-k thoughts?

Reply:
My suggestion:
"With this Renderer Feature, Unity can project specific Materials (decals) onto other objects in the Scene."

Generally, I like the idea of adding tech details to tooltips, but if the renderer feature adds more than render passes, then we should list those things as well, and then this might get complicated.
I might need to describe which passes it adds in the docs though.

Reply:
information is there but i think wording it needs to be 
"Injects a render pass that allows for the rendering of decals to occur"

Reply:
fixed.